lib/babe/epoch_handler.go

```diff
@@ -47,6 +47,9 @@ func newEpochHandler(epochDescriptor *epochDescriptor, constants constants,
 	}
 
 	return &epochHandler{
+		// TODO: we should keep the last slot number from the previous epoch
```

Comment:
Are you going to address it in this PR? If not, could you create an issue to keep track of this TODO please? 🙌

Reply:
Not yet, this annotation was an assumption of mine with regards to changing from one epoch to another, but it works w/o knowing prev epoch last slot number